pkg/csi/networkBuilder.go

```diff
@@ -268,10 +268,7 @@ func createNetFromLayoutConfig(conf NetworkLayoutConfiguration) (*IPV4Network, e
 			}
 			subnet.FullName = fmt.Sprintf("%v Bootstrap DHCP Subnet", tempNet.Name)
 			if tempNet.Name == "NMN" || tempNet.Name == "HMN" || tempNet.Name == "CMN" || tempNet.Name == "CAN" || tempNet.Name == "CHN" {
```

Comment:
Since we no longer care about it, should probably remove `|| tempNet.Name == "CMN"` here.

Reply:
I tried that but then we lose the kube vip IP.  If you missed it and I missed it, then it's a sign of really bad if/then statements.

Reply:
yikes.